gnome/garagejam/Portfile

```diff
@@ -75,3 +69,24 @@ if {![variant_isset x11]} {
 if {![variant_isset quartz] && ![variant_isset x11]} {
     error "Either +x11 or +quartz is required"
 }
+
+# Ensure dbus is loaded after installation
+post-activate {
+  ui_msg "Loading D-Bus service..."
+  system "sudo port load dbus"
```

Comment:
I'm not too sure that a port should port should be doing this automatically... probably better to only ask the user to do so in the notes.  CC: @ryandesign

Reply:
It definitely should not do this.